.nvd-suppressions.xml

```diff
@@ -27,6 +27,13 @@
     <filePath regex="true">.*</filePath>
     <cve>CVE-2022-40152</cve>
   </suppress>
+  <suppress>
+    <notes>
+      No known exploit scenario.
```

Comment:
Please add a little more detail; what library is this about, why are we not vulnerable?